Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby '2.7.0'
+ruby '2.7.2'
```

Comment:
Heads-up - this change affects all contributors' environments. Good to move forward, but might cause a ripple for some.

Reply:
I'll just remove it